<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wVolueBar</class>
 <widget class="QFrame" name="wVolueBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>90</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="widgetBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>561</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,12,1">
    <item>
     <widget class="QLabel" name="typeBar">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>34</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: #33CC99;</string>
      </property>
      <property name="text">
       <string>U</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="frameBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="shadowBar" native="true">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>17</y>
         <width>61</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 100);
border-right:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-bottom:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-left:15px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-top:15px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
background-clip:padding;
/* border-radius: 5px;*/</string>
       </property>
      </widget>
      <widget class="QWidget" name="placeBar" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>61</width>
         <height>391</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #cccccc;</string>
       </property>
       <widget class="QWidget" name="curHBar" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>60</width>
          <height>80</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #33CC99;</string>
        </property>
       </widget>
       <widget class="QWidget" name="amplBarl" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>79</y>
          <width>60</width>
          <height>211</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #66ffcc;</string>
        </property>
       </widget>
       <widget class="QLabel" name="volBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>60</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Proxima Nova Rg</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>13.68</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <zorder>amplBarl</zorder>
       <zorder>curHBar</zorder>
       <zorder>volBar</zorder>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mesBar">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Proxima Nova Rg</family>
        <pointsize>34</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(24, 188, 153);</string>
      </property>
      <property name="text">
       <string>V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
